pulsar-client-cpp/tests/BasicEndToEndTest.cc

```diff
@@ -99,7 +100,7 @@ static void sendCallBackWithDelay(Result r, const Message& msg, std::string pref
 
 class EncKeyReader : public CryptoKeyReader {
    private:
-    void readFile(std::string fileName, std::string& fileContents) const {
+    void readFile(std::string fileName, std::string &fileContents) const {
```

Comment:
`const std::string& fileName`

Reply:
This is existing code and it was just one space reformatted by `make format`. 
In any case the functional change should be done in a different PR. Given this is for test, I'm not particular concerned about a string copy when loading the key file.